doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,232 @@
++                  +--------------------------------+--------------------------------+
+|   Strengths      | Better roaming agreements      | Power efficient at lower data  |
+|                  |                                | rate                           |
++                  +--------------------------------+--------------------------------+
+|                  | Power efficient at medium data |                                |
+|                  | rate                           |                                |
++                  +--------------------------------+--------------------------------+
+|                  | Suitable for TCP/TLS secure    |                                |
+|                  | connection                     |                                |
++------------------+--------------------------------+--------------------------------+
+
+You can find the key feature comparison between LTE-M and NB-IoT in the `nWP044 - Best practices for cellular IoT development`_ documentation.
+
+Factors to be considered for LTE
+********************************
+
+Following are some key aspects to ensure LTE support for your SiP using Nordic Semiconductor's devices, such as the nRF91 Series.
+
+LTE bands
+=========
+
+There are 56 LTE bands from 400 MHz to 6 GHz.
+Some bands use :term:`Frequency Division Duplex (FDD)` and some use :term:`Time Division Duplex (TDD)`.
+The different bands are licensed by different Mobile Network Operators (MNO) around the world.
+Half-duplex operation opens for much simpler multiband support and enables :term:`User equipment (UE)` to operate in a larger region.
+For LTE-M and NB-IoT, the only known deployment is FDD.
+
+nRF91 Series SiPs support up to 18 LTE bands for global operation.
+For details of LTE band support in an nRF91 Series SiP for LTE-M and NB-IoT, see `nRF9161 Product Specification`_ or `nRF9160 Product Specification`_, depending on the SiP you are using.
+
+Physical layer parameters
+-------------------------
+
+The following table provides physical layer parameters for LTE-M and NB-IoT:
+
++----------------------------------+----------------------------------+----------------------------------+
+|                                  | LTE-M                            | NB-IoT                           |
++==================================+==================================+==================================+
+| Bandwidth                        | 1.4 MHz                          | 200 kHz                          |
++----------------------------------+----------------------------------+----------------------------------+
+| Antenna Techniques               | Single antenna                   |  Single antenna                  |
++----------------------------------+----------------------------------+----------------------------------+
+| OFDM subcarriers                 |  OFDM subcarriers (72)           | OFDM subcarriers (12)            |
```

Comment:
In my opinion this does not provide useful additional information to the reader of this document. Number of subcarriers is just another way of indicating the available bandwidth which is already mentioned above.

But if you want to keep it, please remove the repetition of "OFDM Subcarriers" (now it's threre 3 times)